roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -2,16 +2,11 @@
 - name: Create NextFlow directory
   file: path="{{ nextflow_dest }}" state=directory mode=g+rwXs
 
-- name: Put NextFlow installation files in same directory
-  shell: "export NXF_HOME={{ nextflow_dest }}"
-
-- name: Add NextFlow installation files to env var NFX_HOME
-  lineinfile: dest="{{ ngi_pipeline_conf }}/{{ bash_env_script }}"
-              line="export NXF_HOME={{ nextflow_dest }}"
-              backup=no
-
 - name: Install NextFlow
   shell: "./nextflow"
+  environment:
```

Comment:
Using `environment` will set these environment variables /during/ the execution of the playbook. Why is this necessary here? I can't see you referencing these anywhere, nor any steps where they will be implicitly picked up by some step.